### PR TITLE
remove db dependency from config vars

### PR DESCRIPTION
**Pull request by apurva3000**

### CODE REVIEW

File: production_vars_example_extended.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (Jyrsa) [inline production_vars_example_extended.yml]: should we just merge this with the regular _example.yml ?

File: roles/common/templates/etc/pebbles/app_config.j2 [hunk 2]
```diff
@@ -14,7 +15,51 @@ BASE_URL: 'https://{{ public_ipv4 }}/'
...
+FAKE_PROVISIONING = False
+{% endif %}
+{% if skip_task_queue | d() | bool %}
+SKIP_TASK_QUEUE = False
+{% endif %}
+{% if write_provisioning_logs | d() | bool %}
+WRITE_PROVISIONING_LOGS = True
+{% endif %}
+
+{%  if public_ipv4 is defined %}
```
Comment (Jyrsa) [inline roles/common/templates/etc/pebbles/app_config.j2]: i'm not happy with the public_ipv4, especially as it is not intuitively named (i think it takes a FQDN) but let's leave that for later

File: production_vars_example_extended.yml (hunk 1 above)
Comment (apurva3000) [inline production_vars_example_extended.yml]: actually we can

**Comment by Jyrsa** [comment]
This is OK but as discussed let's wait until we merge the other one so we don't have to point to a non-master branch.

**Comment by Jyrsa** [comment]
Okay, not waiting to merge any more

**Pull request merged**